_config.yml

```diff
@@ -35,3 +35,8 @@ gh_edit_repository: 'https://github.com/microsoft/onnxruntime' # the github URL
 gh_edit_branch: 'gh-pages' # the branch that your docs is served from
 # gh_edit_source: docs # the source that your files originate from
 gh_edit_view_mode: 'tree' # "tree" or "edit" if you want the user to jump into the editor immediately
+nav_external_links:
```

Comment:
What impact does this have?

Reply:
It creates a new nav link in the blogs, linking to the GH docs: 
![image](https://github.com/microsoft/onnxruntime/assets/24942306/00ed2b90-3aff-4544-ab7d-0e0137e56822)

Reply:
I don't think we need this. The docs on ONNX Runtime are for contributors. There might be some clean up we need to do on GitHub to make this distinction clear (as the docs all used to be in GitHub only)

Reply:
@faxu can you comment on this change?

Reply:
I think its ok to have it for easy access. While the goal would be to clean up the docs on GH, this help with discoverability.